Description: Upstream changes introduced in version 4.1-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 sml-mode (4.1-1) experimental; urgency=low
 .
   * New upstream version.
   * Removed all autogenerated files.
 .
 The person named in the Author field signed this changelog entry.
Author: Jens Peter Secher <jps@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- sml-mode-4.1.orig/sml-mode-startup.el
+++ sml-mode-4.1/sml-mode-startup.el
@@ -4,7 +4,7 @@
              (or (file-name-directory load-file-name) (car load-path)))
 
 ;;;### (autoloads (sml-yacc-mode sml-lex-mode sml-cm-mode sml-mode)
-;;;;;;  "sml-mode" "sml-mode.el" (19344 19039))
+;;;;;;  "sml-mode" "sml-mode.el" (19595 34277))
 ;;; Generated autoloads from sml-mode.el
 
 (add-to-list 'auto-mode-alist '("\\.s\\(ml\\|ig\\)\\'" . sml-mode))
@@ -39,7 +39,7 @@ Major Mode for editing ML-Yacc files.
 
 ;;;***
 
-;;;### (autoloads nil "sml-proc" "sml-proc.el" (19344 5059))
+;;;### (autoloads nil "sml-proc" "sml-proc.el" (19595 34277))
 ;;; Generated autoloads from sml-proc.el
 
 (autoload 'run-sml "sml-proc" nil t)
@@ -47,7 +47,7 @@ Major Mode for editing ML-Yacc files.
 ;;;***
 
 ;;;### (autoloads nil nil ("sml-compat.el" "sml-defs.el" "sml-move.el"
-;;;;;;  "sml-util.el") (19344 21563 982091))
+;;;;;;  "sml-util.el") (19595 35954 70351))
 
 ;;;***
 
